### DISCUSSION TITLE
Using Magick on a Linux environment When converting PDF files to Png images using NET, the resulting image is blank without any error. Seek help!

### DISCUSSION THREAD

**Discussion by smuannn** [dlemstra/Magick.NET#1632] [Help]
Using Magick on a Linux environment When converting PDF files to Png images using NET, the resulting image is blank without any error. Seek help!
![1](https://github.com/dlemstra/Magick.NET/assets/77477716/91b86d16-9f13-4cee-a47c-2c6497b8abc1)
![2](https://github.com/dlemstra/Magick.NET/assets/77477716/63cdb879-0287-433e-969f-4f34ad12ddfc)
![3](https://github.com/dlemstra/Magick.NET/assets/77477716/96e562b2-1148-4de3-91a3-27f4ddc058e5)

**Comment by dlemstra** [comment]
Do you have Ghostscript installed on your system? You will need to install that if you want to read pdf files.
